src/main/include/Mag.h

```diff
@@ -29,7 +29,7 @@ class Mag : public behaviour::HasBehaviour {
     MagConfig _config;
     MagState _state;
     units::volt_t _voltage;
```

Comment:
if this is what you're using for the raw state you should change this to _rawVoltage and give It a default value (0_V)